specifyweb/frontend/js_src/lib/localization/merging.ts

```diff
@@ -61,17 +61,14 @@ export const mergingText = createDictionary({
     'en-us': 'Dismiss from merging',
   },
   agentContainsGroupDescription: {
-    'en-us': `Agents contain group members`,
+    'en-us': `Agent contain group members`,
```

Comment:
either `Agents contain group members` or `Agent contains group members`. otherwise not gramatically correct

`Agents contain group members` is probably better as it counts for more than one agent having group members